src/testdir/test_vim9_class.vim

```diff
@@ -2331,6 +2331,55 @@ def Test_call_method_in_extended_class()
   v9.CheckScriptSuccess(lines)
 enddef
 
+def Test_instanceof()
```

Comment:
Can you also add a test to the test_vim9_builtin.vim file which passes different types of arguments to the new instanceof() function?